miner/worker.go

```diff
@@ -39,7 +40,7 @@ import (
 
 const (
 	// resultQueueSize is the size of channel listening to sealing result.
-	resultQueueSize = 10
+	resultQueueSize = 20
```

Comment:
how was this number calculated?

Reply:
This variable was increased out of an abundance of caution, since the task queue filling up would cause all of the workers to block. I don't think we're actually close to hitting this, and maybe it should actually be based on the `maxMergedBundles` param, but 20 should be more than enough.